clang/lib/Sema/SemaAPINotes.cpp

```diff
@@ -511,6 +511,10 @@ static void ProcessAPINotes(Sema &S, FunctionOrMethod AnyFunc,
       AnyTypeChanged = true;
   }
 
+  // returns_(un)retained
+  if (!Info.SwiftReturnOwnership.empty())
```

Comment:
I don't think this is a good idea. This lets the users create arbitrary SwiftAttr, even things we do not want them to. This issue goes away if we use enums.